### PR TITLE
Switched to Bunyan, log json in production

### PR DESCRIPTION
**Pull request by serge1peshcoff**

### CODE REVIEW

File: lib/cron.js [hunk 1]
```diff
@@ -93,8 +97,11 @@ class JobManager {
         } = jobType;
 
         if (moment().isAfter(time)) {
-            logger.warn(`Job "${description}" with params %o is not added: \
-is in the past (${moment(time).format('YYYY-MM-DD HH:mm:SS')}), not scheduling.`, params);
+            logger.warn({
+                description,
+                scheduled_on: moment(time).format('YYYY-MM-DD HH:mm:SS'),
```
Comment (linuxbandit) [inline lib/cron.js]: we don't need any timestamp i presume, tbh - the moment when ES receives the log should be the same that `moment` is telling us (for our case, the approximation of ignoring the transmission time of the message is enough imho)

File: lib/logger.js [hunk 2]
```diff
@@ -1,28 +1,11 @@
...
-            format: winston.format.combine(
-                winston.format.colorize(),
-                winston.format.timestamp(),
-                winston.format.align(),
-                winston.format.splat(),
-                winston.format.printf(info => `${info.timestamp} [${info.level}]: ${info.message}`),
-            )
-        })
-    ]
+const logger = bunyan.createLogger({
```
Comment (linuxbandit) [inline lib/logger.js]: what is the difference between the two? the encoding?

Comment (linuxbandit) [reply lib/logger.js]: rephrase: is there really no way to use winston with json, i guess? in which case, it's too much of a hassle to use one driver for dev and one for prod unfortunately

Comment (serge1peshcoff) [reply lib/logger.js]: Main difference:
- bunyan always logs as json and you can pipe it to its cli to make it human readable
- winston love to either json or human readable format, depending on the config, there's no cli.

So in bunyan I can do the following: on dev, log as json and pipe it to cli to get the human readable format. In winston I can do the following: on dev log as human readable format, on prod log as json. But this is less flexible as I can't pipe winston's output to cli on prod (if there is a reason to do it) to get human readable format from terminal. I can do it with bunyan though, that's why I considered switching to it.

File: lib/cron.js (hunk 1 above)
Comment (serge1peshcoff) [reply lib/cron.js]: It's not the timestamp, it's the time a job will be executed at

File: lib/logger.js (hunk 2 above)
Comment (serge1peshcoff) [reply lib/logger.js]: We also use bunyan at work and had no issues with it

Comment (linuxbandit) [reply lib/logger.js]: thanks for the thorough answer, glad we can do what we said

**Pull request merged**